secrets/localsecrets/localsecrets.go

```diff
@@ -24,14 +32,57 @@ package localsecrets // import "gocloud.dev/secrets/localsecrets"
 import (
 	"context"
 	"crypto/rand"
+	"encoding/base64"
 	"errors"
+	"fmt"
 	"io"
+	"net/url"
 
 	"gocloud.dev/gcerrors"
 	"gocloud.dev/secrets"
 	"golang.org/x/crypto/nacl/secretbox"
 )
 
+func init() {
+	secrets.DefaultURLMux().RegisterKeeper(SchemeString, &URLOpener{})
+	secrets.DefaultURLMux().RegisterKeeper(SchemeBase64, &URLOpener{base64: true})
+}
+
+// SchemeString is the URL scheme localsecrets registers its URLOpener under on secrets.DefaultMux.
```

Comment:
nit: Combine these two consts, and have one comment line on top. That way it would show up in the godoc together instead of side by side with the identical documentation.

Reply:
Done.